src/main/java/model/LottoNumber.java

```diff
@@ -1,16 +1,38 @@
 package model;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
 public class LottoNumber implements Comparable<LottoNumber> {
+    public static final int LOTTO_NUMBER_LOWER_BOUND = 1;
+    public static final int LOTTO_NUMBER_UPPER_BOUND = 45;
+    private static final List<LottoNumber> CACHE = new ArrayList<>();
+
+    static {
+        for (int i = LOTTO_NUMBER_LOWER_BOUND; i <= LOTTO_NUMBER_UPPER_BOUND; i++) {
+            CACHE.add(new LottoNumber(i));
+        }
+    }
```

Comment:
아래와 같이 조금 더 깔끔하게 선언해볼 수도 있을 것 같네요. 😉

```suggestion
    private static final List<LottoNumber> CACHE;

    static {
            CACHE = IntStream.rangeClosed(LOTTO_NUMBER_LOWER_BOUND, LOTTO_NUMBER_UPPER_BOUND)
                .mapToObj(LottoNumber::new)
                .collect(Collectors.toList());
    }
```

Reply:
IntStream을 사용하여 변경했습니다. [d4f3907](https://github.com/next-step/java-lotto-kakao/pull/36/commits/d4f39070f69fd39b634b88f803f63a405af59857)